test/distributed/checkpoint/e2e/test_pipeline.py

```diff
@@ -90,7 +90,7 @@ def load_with_fsdp(self, pipeline_dir: str) -> None:
             optim_state_dict=optim_state_dict,
         )
 
-    @skip_if_lt_x_gpu(4)
+    @skip_if_lt_x_gpu(8)
```

Comment:
Just curious, do multi-gpu CI tests have 8 gpus? Does 4 not enough for this test case?

Reply:
oh, right, I was trying something else. Let me fix this.